models/articles.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../db/connection')
+exports.fetchArticles = (article_id) =>{
+
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`,[article_id])
+    .then(({rows})=>{
+       
+        if(!rows.length){
+            return Promise.reject({status: 404, msg:"Not found"})
+        }
+        return rows
+    })
+    .catch((err)=>{
+        
+        return Promise.reject(err)
```

Comment:
You shouldn't need to catch the error here - it can be caught in the controller and then bounced to the error handlers via next